src/adios4dolfinx/legacy_readers.py

```diff
@@ -353,14 +318,37 @@ def read_mesh_from_legacy_checkpoint(
         gdim=mesh_geometry.shape[1],
     )
     domain = ufl.Mesh(element)
-
     return dolfinx.mesh.create_mesh(
         MPI.COMM_WORLD, mesh_topology, mesh_geometry, domain
     )
 
+# Function for reading global cells. Might be needed later
```

Comment:
Lets put this function in an issue, with some context;)

Reply:
Removed function in https://github.com/jorgensd/adios4dolfinx/pull/22/commits/6bbe5dd885852cf87d5622d5bccaa07b1b39f4be and created issue here https://github.com/jorgensd/adios4dolfinx/issues/23